tools/scripts/bumpenvs.yaml

```diff
@@ -1,117 +1,123 @@
 ap_northeast_1_agent_ami:
-  new: ami-0a01b67743d465a6e
-  old: ami-07dc1e0617119df61
+  new: ami-0881b60d3da7bd055
+  old: ami-0a01b67743d465a6e
 ap_northeast_1_bastion_ami:
-  new: ami-05855b7c54f91a1ae
-  old: ami-0f475b2993ac825b7
+  new: ami-07faf6b16c502e9e1
+  old: ami-05855b7c54f91a1ae
 ap_northeast_1_master_ami:
-  new: ami-0af17d43b1f4ff4ea
-  old: ami-0b8cd500f483d09ee
+  new: ami-03ec47ea54612a98d
+  old: ami-0af17d43b1f4ff4ea
 ap_northeast_2_agent_ami:
-  new: ami-0c74312df78509701
-  old: ami-0a0061c1a3f27fe51
+  new: ami-045eb8957d930ee48
+  old: ami-0c74312df78509701
 ap_northeast_2_bastion_ami:
-  new: ami-063acefae626a1e95
-  old: ami-0eebe903cd90a3ddd
+  new: ami-0ee428c63274f56c1
+  old: ami-063acefae626a1e95
 ap_northeast_2_master_ami:
-  new: ami-04e6fcf8cfe3b09ea
-  old: ami-07b7be0099924913e
+  new: ami-00813ad98dc81f443
+  old: ami-04e6fcf8cfe3b09ea
 ap_southeast_1_agent_ami:
-  new: ami-0eddf3f941d216f11
-  old: ami-0994ac61987afed77
+  new: ami-0b5cca90cd338e59d
+  old: ami-0eddf3f941d216f11
 ap_southeast_1_bastion_ami:
-  new: ami-0311dbbdd981577d5
-  old: ami-03060465516794b47
+  new: ami-06bcc4026e190b380
+  old: ami-0311dbbdd981577d5
 ap_southeast_1_master_ami:
-  new: ami-0ba45f1ec3dff60f9
-  old: ami-028a5cff2f5a0f6c3
+  new: ami-0acac505592775345
+  old: ami-0ba45f1ec3dff60f9
 ap_southeast_2_agent_ami:
-  new: ami-0c54777eb400a9e68
-  old: ami-0183cafc9f2a57aa6
+  new: ami-0725e68bb3b036074
+  old: ami-0c54777eb400a9e68
 ap_southeast_2_bastion_ami:
-  new: ami-0dea9aeaa95b9751b
-  old: ami-001dbdb48850e6f54
+  new: ami-0b079c5b689b2d44e
+  old: ami-0dea9aeaa95b9751b
 ap_southeast_2_master_ami:
-  new: ami-004d0fb87d10716c6
-  old: ami-041e1cc8f4c429789
+  new: ami-07c5b4ba4c9dd25a3
+  old: ami-004d0fb87d10716c6
+cuda_11_hashed:
+  new: determinedai/environments:cuda-11.0-pytorch-1.7-tf-2.4-gpu-5f6f6e1
+  old: __NON-EXISTANT-IMAGE__
```

Comment:
I think we should prefer to omit the `old` field rather than put a string in there.